lib/order_repository.rb

```diff
@@ -0,0 +1,122 @@
+require_relative "./order"
+require_relative "./item"
+require_relative "./shared_methods"
+
+class OrderRepository
+
+  #-----------------
+  # ALL
+  #-----------------
+
+  def all
+    sql = 'SELECT * FROM orders;'
+    result = DatabaseConnection.exec_params(sql, [])
+    
+    orders = []
+    result.each { |record| orders << Record.to_order(record) } # method in ./shared_methods.rb
+    orders.map! { |order| find(order.id) } # get items for order.items
+
+    return orders # return orders with their respective items
```

Comment:
Similar remark here than in a few other places - the comment adds a bit of redundancy on the code intent. Most of your code is quite readable, so adding the extra comment is not needed and can even add a bit to the cognitive load if someone needs to read the code. Maybe renaming the method `all_with_items` would express better that the respective items are also returned, removing then the need for a comment?